dissect/target/loaders/velociraptor.py

```diff
@@ -136,9 +136,14 @@ def map(self, target: Target) -> None:
 
 class VelociraptorDirectoryFilesystem(DirectoryFilesystem):
     def _resolve_path(self, path: str) -> Path:
-        return super()._resolve_path(quote(path, "$/").replace(".", "%2E"))
+        path = quote(path, "$/")
```

Comment:
```suggestion
        path = quote(path, safe="$/")
```
Had to lookup what the second argument was, don't think it's commonly known.

Reply:
Fixed